static/js/compose_actions.js

```diff
@@ -141,8 +141,16 @@ function composing_to_current_topic_narrow() {
     );
 }
 
+function composing_to_current_pm_narrow() {
+    return util.lower_same(
+        compose_state.private_message_recipient(),
+        narrow_state.pm_emails_string() || "",
```

Comment:
How did you check that these two values will have the same format / ordering/etc.?

Reply:
In hindsight this was a naive way to compare this data, in the most recent commit I sort the arrays then stringify them and compare the values. Thank you for pointing this out, I'd be interested to know if there is an even better way :)